index.html

```diff
@@ -98,8 +98,220 @@ <h2>1989</h2>
+									<h3>Dessert 2</h3>
+									<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aperiam nisi debitis nam culpa dicta velit nulla ut et soluta, numquam?</p>
+				    		</div>
+				    		<div class="col-md-3">
+				    			<img src="img/dessert3.jpg" alt="dessert3" class="img-responsive img-thumbnail">
+									<h3>Dessert 3</h3>
+									<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dicta quia, distinctio mollitia molestias natus unde dolores magni similique aliquid eum!</p>
+				    		</div>
+				    		<div class="col-md-3">
+				    			<img src="img/dessert4.jpg" alt="dessert4" class="img-responsive img-thumbnail">
+									<h3>Dessert 4</h3>
+									<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quia saepe laboriosam reprehenderit eaque accusantium recusandae, doloremque enim eveniet debitis sequi!</p>
+				    		</div>
+				    	</div>
+				    </div>
+				  </div>
+				</div>
+			</div>
+		</div>
+	</div>
+	<!--END MENU-->
+	<!--START CONTACT -->
+		<div class="container-fluid" id="contact">
+			<div class="container">
+				<div class="row">
+					<div class="col-md-12">
+						<h1 class="text-uppercase">Contact Us</h1>
+						<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Cumque voluptatem sit doloribus nostrum maxime accusantium quo odio vero sunt, earum inventore quis veritatis repudiandae officiis nisi reprehenderit modi accusamus magni.</p>
+					</div>
+				</div>
+				<div class="row">
+					<div class="col-md-6">
+							<form>
+								<div class="form-group">
+							    <label for="exampleInputName">Name <span>*</span></label>
+							    <input type="text" class="form-control" id="exampleInputName" placeholder="Name">
+							  </div>
+								<div class="form-group">
+							    <label for="exampleInputEmail1">Email <span>*</span></label>
+							    <input type="email" class="form-control" id="exampleInputEmail1" placeholder="Email">
+							  </div>
+							  <div class="form-group">
+							    <label for="exampleInputSubject">Subject <span>*</span></label>
+							    <input type="text" class="form-control" id="exampleInputSubject" placeholder="Subject">
+							  </div>
+							  <div class="form-group">
+							    <label for="exampleInputMessage">Message <span>*</span></label>
+							    <textarea class="form-control" name="name" rows="6"></textarea>
+							  </div>
+							  <div class="checkbox">
+							    <label>
+							      <input type="checkbox"> I'm not a robot.
+							    </label>
+							  </div>
+							  <button type="submit" class="btn btn-default center-block">Send Message</button>
+								<p class="text-center" id="fine-print">All fields marked
+								<span>*</span>are mandatory.</p>
+							</form>
+					</div>
+					<div class="col-md-6">
+						<address>
+							<a class="name" href="#">
```

Comment:
Why are you using "a" tags for the contact information? Is there something special you want to link them to? (It's definitely possible I missed Rocky talking about this lol)

Reply:
Here's the HTML I used: 
< address>
	< div class="name">
		< stong>RESTAURANTE< /strong>< br >
	< /div>
	< div class="street">
		123 Fulton St. < br>
	< /div>
	< div class="city">
		Fresno, CA 93721 < br>
	< /div>
	< div class="phone">
		P : (123) 456-7890
	< /div>
< /address>

*Disclaimer: I'm confirming my code with Rocky, so still waiting for a response.

Reply:
I am curious about the address. I wrapped my code in a p tag. Do we need the extra tags and classes?